manifests/3/3CX/Softphone/18.12.425/3CX.Softphone.installer.yaml

```diff
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.2.4 $debug=QUSU.CRLF.5-1-19041-2673.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: 3CX.Softphone
+PackageVersion: 18.12.425
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Scope: machine
```

Comment:
```suggestion
Scope: user
```

I believe `{77DE5C80-1E23-4C1A-B78C-29BFF0BAAFB4}.msq` should be the primary ARP entry since `winget uninstall --id "{77DE5C80-1E23-4C1A-B78C-29BFF0BAAFB4}"` does not uninstall the package for me. (And the version that appears in `appwiz.cpl` is also 18.12.425 and not 18.12.425.0)
![image](https://user-images.githubusercontent.com/88161975/232638277-7466aa6d-7f7a-4d7e-b705-4eb16b66a57e.png)

And if we were to change the ARP entry, the scope should also be set to `user` since the ARP entry is located in `Computer\HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Uninstall\{77DE5C80-1E23-4C1A-B78C-29BFF0BAAFB4}.msq` and the program installs to `%LOCALAPPDATA%\Programs\3CXDesktopApp`.

(Speaking of Scope: user, have we found a wix installer that installs per-user and writes to the correct registry location of HKEY_USERS? 😮This might be something to mention over at the winget-cli repo related issue)

Reply:
Will fix